src/aap_eda/core/views.py

```diff
@@ -12,6 +12,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+from ansible_base.lib.constants import STATUS_GOOD
```

Comment:
Can you also import STATUS_FAILED? It should be used on https://github.com/ansible/eda-server/pull/1273/files#diff-b7f4c079e5cb3a3c01996199db0ee6474eb5618e60d85a85ac2a08795268908aL60

Reply:
Updated. Thanks.